AIDevGallery/ProjectGenerator/Template/MainWindow.xaml.cs

```diff
@@ -17,4 +20,30 @@ internal void ModelLoaded()
     {
         ProgressRingGrid.Visibility = Visibility.Collapsed;
     }
+
+    internal async void ShowException(Exception ex, string? optionalMessage = null)
+    {
+        var msg = optionalMessage ?? ex switch
+        {
+            COMException
```

Comment:
IMO, we should have this code in the sample itself as this method can be generalized for all samples, not just for WCR samples. The ShowException method should be very general